xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/informationinline.vm

```diff
@@ -20,6 +21,19 @@
         #end
       </dd>
     </dl>
+    <dl>
+      <dt>$services.localization.render('core.viewers.information.pageReference')</dt>
+      <dd id="information-reference">
+        <span class="noitems">$services.localization.render('core.viewers.information.pageReference.tips')</span>
```

Comment:
We put the hint in the ``dt`` when [creating forms](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/#H4.Hint), but I guess it's OK in the ``dd`` also (this is not an actual form).

Reply:
I followed [the snippet](https://jira.xwiki.org/browse/XWIKI-13362?focusedCommentId=101580&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-101580) proposed by @evalica for this one and it looks good. I don't think it's breaking a WCAG rule either, but let me know if you think I should change.